mtp_api/apps/security/management/commands/update_security_profiles.py

```diff
@@ -0,0 +1,107 @@
+from django.db.models import Max
+from django.core.management import BaseCommand
+
+from credit.models import Credit
+from security.models import (
+    SenderProfile, BankTransferSenderDetails, DebitCardSenderDetails,
+    PrisonerProfile, SecurityDataUpdate, CardholderName
+)
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        try:
+            last_updated_at = SecurityDataUpdate.objects.latest().updated_at
+            new_credits = Credit.objects.filter(received_at__gt=last_updated_at)
```

Comment:
might be safer to do `received_at__gte`… in case there are a few at the same moment